src/modules/users/consumers/PermissionsConsumer.spec.ts

```diff
@@ -88,14 +88,20 @@ describe('PermissionsConsumer', () => {
 
       await createUsers(prisma, userIds)
 
-      publisher.publishSync('user_permissions', {
-        id: 'abc',
-        payload: {
-          userIds,
-          permissions: perms,
+      publisher.publishSync(
+        'permissions',
+        {
+          id: 'abc',
+          payload: {
+            userIds,
+            permissions: perms,
+          },
+          type: 'permissions.added',
         },
-        type: 'permissions.added',
-      })
+        {
+          routingKey: '',
```

Comment:
@CarlosGamero Should we make this parameter optional and default to ''? It is mandatory for publish to topic operation in RabbitMQ, but empty value is acceptable.

Reply:
I would say the same as before, if it is a valid value used in real cases I would vote for simplify the API by having it optional